### DISCUSSION TITLE
Allow branches to require all requested reviewers to approve

### DISCUSSION THREAD

**Discussion by abroadwin** [community/community#38616] [Pull Requests]
At the moment it's possible to require code owners or a specific number of people to review a PR before allowing it to be merged. I'd love a third option: require everyone who was requested as a reviewer of a specific PR to sign off before merging.

**Comment by david-engelmann** [comment]
One possible option would be going to Code Owners route. You can add a branch rule to require approval of code owners. With the CODEOWNERS file you can list multiple users, see the snippet below.

```
* @engineer1 @enginer2
```
Or For Certain parts of the code
```
src @engineer1 @enginer2
test @engineer3
```

[Stackoverflow post on the topic](https://stackoverflow.com/questions/66821568/require-github-pr-approvals-from-multiple-groups). Groups could definitely add an organizational layer

**Reply by abroadwin** [reply]
This doesn't address all of my use-cases, though. Consider when one wants a PR to be approved by a person in their organization who requested a change in addition to another engineer. The person who requested a change isn't necessarily someone who belongs on the list of code owners.
